sdk/python/kfp/dsl/types/type_utils.py

```diff
@@ -263,6 +263,16 @@ def verify_type_compatibility(
     expected_type = expected_spec.type
     given_type = _get_type_string_from_component_argument(given_value)
 
+    # avoid circular imports
+    from kfp.dsl.for_loop import LoopArgument
+    from kfp.dsl.for_loop import LoopArgumentVariable
```

Comment:
Can we update this to `from kfp.dsl import for_loop` ([ref](go/pystyle#imports))?